mindmeld/models/model.py

```diff
@@ -350,12 +365,19 @@ def register_resources(self, **kwargs):  # pylint: disable=no-self-use
     def get_resource(self, name):
         return self._resources.get(name)
 
-    @staticmethod
-    def _get_model_config_save_path(path):
-        head, ext = os.path.splitext(path)
-        model_config_save_path = head + ".config" + ext
-        os.makedirs(os.path.dirname(model_config_save_path), exist_ok=True)
-        return model_config_save_path
+    @property
+    def text_preparation_pipeline(self):
+
+        text_preparation_pipeline = self._resources.get("text_preparation_pipeline")
+
+        if not text_preparation_pipeline:
+            logger.error(
+                "The text_preparation_pipeline resource has not been registered "
```

Comment:
This warning may get confusing to developers since they would think it is referring to the 'global' text_preparation_pipeline

Reply:
hey @KKsharma99 this actually refers to global one only. Upon some digging, I found the following snippet in `initialize_resources()` methods in model.py file. So in absence of a global text_preparation_pipeline, a new one is created and hence the error message. (Also, this message already existed in the codebase and is not something that I newly added. )

```
        # Always initialize the global resource for tokenization, which is not a
        # feature-specific resource
        self._resources[
            "text_preparation_pipeline"
        ] = resource_loader.get_text_preparation_pipeline()
```